src/nashpy/algorithms/regret_minimization.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from typing import Generator, Tuple, Any
+import numpy.typing as npt
+
+
+def compute_regrets(strategy_utilities, current_strategy):
+    """
+    This function calculates the regrets for a player based on the strategy utilities and the current strategy.
+    Regrets represent the difference between the utility achieved by playing a strategy and the maximum utility
+    that could have been achieved by playing any strategy.
+    In this implementation, only positive regrets are considered
+    """
+    regrets = np.maximum(0, strategy_utilities - current_strategy)
+    return regrets
+
+
+def update_strategy(current_strategy, regrets, learning_rate):
+    """
+    This function updates the player's strategy based on the regrets, the current strategy, and a fixed learning rate.
+    It scales the regrets by the learning rate and adds them to the current strategy.
+    Finally, it normalizes the updated strategy to ensure that the probabilities sum up to 1.
+    """
+    updated_strategy = current_strategy + learning_rate * regrets
+    return updated_strategy / np.sum(updated_strategy)
+
+
+def generate_abs_strategy(strategy_list):
+    """
+    This function will return most favorable utility by a player based on the max probability value
+    """
+    max_probability = max(strategy_list)
+    strategy_relative = [1 if x == max_probability else 0 for x in strategy_list]
+    sum_value_in_the_list = sum(strategy_relative)
+    favorable_strategy = [x / sum_value_in_the_list for x in strategy_relative]
+    return favorable_strategy
+
+
+def regret_minimization(
+    A: npt.NDArray, B: npt.NDArray, learning_rate=0.1, max_iterations=100
```

Comment:
```suggestion
    A: npt.NDArray, B: npt.NDArray, learning_rate=0.1, iterations=100
```

Reply:
Changed in the latest commit.

Reply:
Also changed, in the test, functions, docs, how-to and in the game code.